### PR TITLE
WIP: Add first version of Wall of Fame

### PR DESCRIPTION
**Pull request by bixel**
based on Maries Document

### CODE REVIEW

**Comment by bixel** [comment]
Ping @marschmitz :-)

**Comment by MaxNoe** [comment]
Why don't we use posts for the wall of fame? Seems to me that this would be more the jekyll way to do it.

We could use a different template and not show them on the main page.

File: ueber_uns/wall_of_fame.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ Alltagsphysik für jeden verständlich erklärt.
 </div>
 
 <div class="box" markdown="1">
-# Christophe Cauet 
-*Gründer und Data Scientist bei Point 8 *  
+# Christophe Cauet
+*Gründer und Data Scientist bei Point 8*
```
Comment (bixel) [inline ueber_uns/wall_of_fame.md]: I didn't even recognize this. Maybe I should use vim again 😕

**Comment by bixel** [comment]
Mh true. I interpreted as a relatively static content but moving to a specific post layout makes sense.

**Comment by bixel** [comment]
@MaxNoe this should be better. I just realised we are doing the same thing for the job offers. I created a specific directory `wall_of_fame/_posts` which jekyll will catch up such that every post in there will be in the `wall_of_fame` category. Do you think this is too confusing? Maybe we can open an issue to discuss that and either create a separate directory for job offers or move all wof-posts into the main post directory and manually categorize them.

**Comment by MaxNoe** [comment]
shouldn't be `_posts/wall_of_fame` also be fine?

**Comment by bixel** [comment]
Thought so too, but found an issue saying that opposite. But I didn't try to be honest.

**Pull request merged**